example/scp.c

```diff
@@ -145,6 +149,10 @@ int main(int argc, char *argv[])
         goto shutdown;
     }
 
+#ifdef WIN32
```

Comment:
I'd love to keep this change separate from the others since it seems not so related to handling > 2GB files.  Perhaps a separate pull request directly against upstream?